projects/1.20/assets/architects-palette/architects_palette/lang/zh_cn.json

```
@@ -0,0 +1,643 @@
+  "architects_palette.info.totem_carving": "在图腾上使用斧以切换图腾样式。",
+  "architects_palette.info.flint_damage": "燧石很硬！生物摔落到燧石方块上时受到的摔落伤害增加25%。",
+  "architects_palette.info.celestial_stones": "这些石头会根据当前时间改变自身形态和亮度。在所有维度中都有效。",
+  "architects_palette.info.heavy_bricks": "每两个相邻方块会连接在一起，便于制造大型砖块状纹理。",
+  "architects_palette.info.cage_lanterns": "可由红石信号点亮。控制模式可通过右击反转。",
+  "architects_palette.info.nether_brass": "下界黄铜被点燃时，其上火焰会呈现出独特的绿色。",
+  "architects_palette.info.wardstone": "青卫石方块会弹开射到其上的弹射物。",
+
+  "architects_palette.advancement.root": "建筑师的调色板",
+  "architects_palette.advancement.root.desc": "是时候建点新的东西了。",
+  "architects_palette.advancement.totem_carving": "让你笑不出来",
+  "architects_palette.advancement.totem_carving.desc": "用斧将一个金合欢木图腾雕刻成另一种样式。",
+  "architects_palette.advancement.buy_fish_block": "好大一条鱼！",
+  "architects_palette.advancement.buy_fish_block.desc": "与渔夫村民交易得到一块新鲜的鱼肉。",
+  "architects_palette.advancement.buy_pipe": "本地的管道工",
+  "architects_palette.advancement.buy_pipe.desc": "从工具匠手里买下一些管道方块。",
+  "architects_palette.advancement.buy_entrails": "精细地屠宰",
+  "architects_palette.advancement.buy_entrails.desc": "用你宝贵的绿宝石换一块……肠子？",
+  "architects_palette.advancement.place_entrails": "不知怎么地，比想象的更糟",
+  "architects_palette.advancement.place_entrails.desc": "放下并注视你新买回来的肠子。",
+  "architects_palette.advancement.buy_plating": "厚厚的钢板",
+  "architects_palette.advancement.buy_plating.desc": "与盔甲匠村民交易得到一些他们的金属板",
+  "architects_palette.advancement.buy_spool": "批量采购",
+  "architects_palette.advancement.buy_spool.desc": "买一卷牧羊人的羊毛，或者你自己合成一卷。",
+  "architects_palette.advancement.buy_celestial_stone": "想买个钟表吗？",
+  "architects_palette.advancement.buy_celestial_stone.desc": "从流浪商人手中购买一些日长石或者月长石。",
+  "architects_palette.advancement.find_warpstone": "这是什么东西？",
+  "architects_palette.advancement.find_warpstone.desc": "将一个黏土块扔过下界传送门，然后在传送门的另一侧发现不一样的东西。",
+  "architects_palette.advancement.find_twisted_sapling": "现实扭曲",
+  "architects_palette.advancement.find_twisted_sapling.desc": "将一个普通树苗移形为扭曲树苗。",
+
+  "block.architects_palette.abyssaline": "渊晶石",
+  "block.architects_palette.abyssaline_bricks": "渊晶石砖",
+  "block.architects_palette.chiseled_abyssaline_bricks": "錾制渊晶石砖",
+  "block.architects_palette.abyssaline_pillar": "渊晶石柱",
+  "block.architects_palette.abyssaline_brick_stairs": "渊晶石砖楼梯",
+  "block.architects_palette.abyssaline_brick_slab": "渊晶石砖台阶",
+  "block.architects_palette.abyssaline_brick_vertical_slab": "渊晶石砖竖直台阶",
+  "block.architects_palette.abyssaline_brick_wall": "渊晶石砖墙",
+  "block.architects_palette.abyssaline_tiles": "渊晶石瓦",
+  "block.architects_palette.abyssaline_tile_stairs": "渊晶石瓦楼梯",
+  "block.architects_palette.abyssaline_tile_slab": "渊晶石瓦台阶",
+  "block.architects_palette.abyssaline_tile_vertical_slab": "渊晶石瓦竖直台阶",
+  "block.architects_palette.abyssaline_lamp": "渊晶石灯",
+
+  "block.architects_palette.salmon_log": "鲑鱼块",
+  "block.architects_palette.cod_log": "鳕鱼块",
+  "block.architects_palette.salmon_scales": "鲑鱼鳞片",

```

```suggestion
  "block.architects_palette.salmon_scales": "鲑鱼鳞片块",
```
虽然没有实际下载并游玩，但这里的key似乎是block为前缀，是否应该译为方块？如采纳，还请全文通查